src/components/button/test-button.js

```diff
@@ -88,24 +88,17 @@ describe('Button', function() {
     assert.equal(wrapper.text(), 'Hello, world.');
   });
 
-  it('sets the type class', function() {
+  it('sets the type attribute', function() {
     const wrapper = renderComponent({
-      type: 'neutral'
+      type: 'reset'
     });
-    assert.equal(wrapper.prop('className'), 'button--neutral');
```

Comment:
With these test changes I don't see anywhere that we're actually testing the rendered output of the component.

Reply:
Added a snapshot